src/components/dialog/dialog.css

```diff
@@ -1,4 +1,21 @@
+@import '../app/variablesV2.css';
+@import '../app/mixins.css';
+
 .dialog {
+  border-radius: 6px;
+
+  & .title {
+    @mixin headingNormalIntermediate;
```

Comment:
I just checked with Domenico, and we shouldn't need this mixin, since it's not on the styleguide, we should or use the large or the normal, just check with him in each case.